heroic/hc_coven_of_shivarra.simc

```diff
@@ -1,5 +1,33 @@
 # movement
+# Whirling Saber
+raid_events+=/movement,cooldown=30,duration=1,first=10,last=200,players_only=1,player_chance=0.2
+raid_events+=/movement,cooldown=30,duration=1,first=400,players_only=1,player_chance=0.2
+# Storm of Darkness
+# could spawn one under you, so not totally 100%
+raid_events+=/movement,cooldown=60,duration=3,first=30,last=330,players_only=1,player_chance=0.8
 
 # adds
+# Machinations of Aman'Thul - 4
+# assume two adds die in 20s, other two take 40s
+raid_events+=/adds,count=1,first=95,duration=20,cooldown=5000,max_distance=30
+raid_events+=/adds,count=1,first=95,duration=20,cooldown=5000,max_distance=30
+raid_events+=/adds,count=1,first=95,duration=40,cooldown=5000,max_distance=30
+raid_events+=/adds,count=1,first=95,duration=40,cooldown=5000,max_distance=30
+# Fury of Golganneth - 4
+raid_events+=/adds,count=1,first=180,duration=20,cooldown=5000,max_distance=30
+raid_events+=/adds,count=1,first=180,duration=20,cooldown=5000,max_distance=30
+raid_events+=/adds,count=1,first=180,duration=20,cooldown=5000,max_distance=30
+raid_events+=/adds,count=1,first=180,duration=30,cooldown=5000,max_distance=30
+# Flames of Khaz'goroth - spawn 8 assume 6 in range
+raid_events+=/adds,count=1,first=270,duration=15,cooldown=5000,max_distance=30
+raid_events+=/adds,count=1,first=270,duration=15,cooldown=5000,max_distance=30
+raid_events+=/adds,count=1,first=270,duration=20,cooldown=5000,max_distance=30
+raid_events+=/adds,count=1,first=270,duration=25,cooldown=5000,max_distance=30
+raid_events+=/adds,count=1,first=270,duration=25,cooldown=5000,max_distance=30
+raid_events+=/adds,count=1,first=270,duration=30,cooldown=5000,max_distance=30
```

Comment:
Decided not to add the Norganon adds, only useful for the sephuz proc as far as a sim is concerned